libs/template/presets/b2b-storefront/app.ts

```diff
@@ -3,10 +3,9 @@ import { provideExperienceData } from '@spryker-oryx/experience';
 import { storefrontFeatures } from '@spryker-oryx/presets/storefront';
 import { contactPage } from './experience/contact-page';
 import { HeaderTemplate } from './experience/header';
-import { cartsPage } from './experience/pages';
 
 const StaticB2BExperienceFeature: AppFeature = {
-  providers: [provideExperienceData([contactPage, HeaderTemplate, cartsPage])],
+  providers: [provideExperienceData([contactPage, HeaderTemplate])],
```

Comment:
But why it's not under feature version?

Reply:
It's not a BC. Page was added as a part of another sub-task of this feature, but in this one I moved it to the main storefront preset